src/Orleans/Configuration/ClientConfiguration.cs

```diff
@@ -175,7 +175,11 @@ public ClientConfiguration()
             NetInterface = null;
             Port = 0;
             DNSHostName = Dns.GetHostName();
+#if NETSTANDARD
+            DeploymentId = Environment.MachineName;
+#else
```

Comment:
Done, rebased since I wanted to keep the commits separate on not force you to squash when merging. Setting this as the empty string was the only change I did since the last time reviewed